### PR TITLE
Add CONTRIBUTING.md file from Wiki

### PR DESCRIPTION
**Pull request by alancutter**
This change pulls the Github Wiki page in as CONTRIBUTING.md.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+   To do this, you'll need to be a collaborator [on the NPM project](https://www.npmjs.com/package/web-animations-js), or have a collaborator help you.
+
+7. If there are any breaking changes to the API in this release you must notify web-animations-changes@googlegroups.com.
+
+   Only owners of the group may post to it so you may need to request ownership or ask someone to post it for you.
+
+## Pushing from web-animations-next to web-animations-js
+
+    git fetch web-animations-next
```
Comment (suzyh) [inline CONTRIBUTING.md]: Missing code block syntax?

**Comment by suzyh** [comment]
LGTM. Also I guess you'll need to update the wiki to point back to this page.

File: CONTRIBUTING.md (hunk 1 above)
Comment (alancutter) [inline CONTRIBUTING.md]: Four space indents make code blocks.

Comment (suzyh) [inline CONTRIBUTING.md]: Ah, found the "rich diff display" button. How confusing.

**Pull request merged**